src/app/views/helpers/css_injector.ts

```diff
@@ -0,0 +1,13 @@
+///<reference path="../../../../typings/browser/ambient/jquery/index.d.ts"/>
+///<reference path="../editors/editor_loader.ts"/>
+
+class CSSInjector {
+  addCSSToHead(url:string): void {
```

Comment:
i'm not sure if it would make sense to make this method private, and test it through loadDropzoneCSS?